docs/framework/wcf/feature-details/migrating-wse-3-0-web-services-to-wcf.md

```diff
@@ -60,7 +60,7 @@ The benefits of migrating WSE 3.0 Web services to Windows Communication Foundati
  For more information about creating custom bindings in WCF, see [Custom Bindings](../../../../docs/framework/wcf/extending/custom-bindings.md).  
   
 ### WSE 3.0 Web services that are secured using application code  
- Whether WSE 3.0 or WCF is used, the security requirements can be specified in application code instead of in configuration. In WSE 3.0, this is accomplished by creating a class that derives from the `Policy` class and then by adding the requirements by calling the `Add` method. For more details about specifying the security requirements in code, see [How to: Secure a Web Service Without Using a Policy File](https://go.microsoft.com/fwlink/?LinkId=73747). In WCF, to specify security requirements in code, create an instance of the <xref:System.ServiceModel.Channels.BindingElementCollection> class and add an instance of a <xref:System.ServiceModel.Channels.SecurityBindingElement> to the <xref:System.ServiceModel.Channels.BindingElementCollection>. The security assertion requirements are set using the static authentication mode helper methods of the <xref:System.ServiceModel.Channels.SecurityBindingElement> class. For more details about specifying security requirements in code using WCF, see [How to: Create a Custom Binding Using the SecurityBindingElement](../../../../docs/framework/wcf/feature-details/how-to-create-a-custom-binding-using-the-securitybindingelement.md) and [How to: Create a SecurityBindingElement for a Specified Authentication Mode](../../../../docs/framework/wcf/feature-details/how-to-create-a-securitybindingelement-for-a-specified-authentication-mode.md).  
+ Whether WSE 3.0 or WCF is used, the security requirements can be specified in application code instead of in configuration. In WSE 3.0, this is accomplished by creating a class that derives from the `Policy` class and then by adding the requirements by calling the `Add` method. For more details about specifying the security requirements in code, see [How to: Secure a Web Service Without Using a Policy File](https://docs.microsoft.com/). In WCF, to specify security requirements in code, create an instance of the <xref:System.ServiceModel.Channels.BindingElementCollection> class and add an instance of a <xref:System.ServiceModel.Channels.SecurityBindingElement> to the <xref:System.ServiceModel.Channels.BindingElementCollection>. The security assertion requirements are set using the static authentication mode helper methods of the <xref:System.ServiceModel.Channels.SecurityBindingElement> class. For more details about specifying security requirements in code using WCF, see [How to: Create a Custom Binding Using the SecurityBindingElement](../../../../docs/framework/wcf/feature-details/how-to-create-a-custom-binding-using-the-securitybindingelement.md) and [How to: Create a SecurityBindingElement for a Specified Authentication Mode](../../../../docs/framework/wcf/feature-details/how-to-create-a-securitybindingelement-for-a-specified-authentication-mode.md).  
```

Comment:
```suggestion
 Whether WSE 3.0 or WCF is used, the security requirements can be specified in application code instead of in configuration. In WSE 3.0, this is accomplished by creating a class that derives from the `Policy` class and then by adding the requirements by calling the `Add` method. For more details about specifying the security requirements in code, see [How to: Secure a Web Service Without Using a Policy File](https://docs.microsoft.com/previous-versions/dotnet/netframework-2.0/aa528763(v=msdn.10)). In WCF, to specify security requirements in code, create an instance of the <xref:System.ServiceModel.Channels.BindingElementCollection> class and add an instance of a <xref:System.ServiceModel.Channels.SecurityBindingElement> to the <xref:System.ServiceModel.Channels.BindingElementCollection>. The security assertion requirements are set using the static authentication mode helper methods of the <xref:System.ServiceModel.Channels.SecurityBindingElement> class. For more details about specifying security requirements in code using WCF, see [How to: Create a Custom Binding Using the SecurityBindingElement](how-to-create-a-custom-binding-using-the-securitybindingelement.md) and [How to: Create a SecurityBindingElement for a Specified Authentication Mode](how-to-create-a-securitybindingelement-for-a-specified-authentication-mode.md).  
```